skipruntime-ts/api/src/api.ts

```diff
@@ -646,7 +646,11 @@ export interface SkipService<
   Inputs extends NamedCollections = NamedCollections,
   ResourceInputs extends NamedCollections = NamedCollections,
 > {
-  /** Initial data for this service's input collections. */
+  /**
+   * Initial data for this service's input collections.
+   *
+   * @remarks While the initial data is not required to have a `DepSafe` type (only a subtype of `Json` is required); note that any modifications made to any objects passed as `initialData` will *not* be seen by a service once started.
```

Comment:
We could instead constraint the type to make this comment unnecessary, but that would force users to call deepFreeze on all their initialData values. I'm unsure, but it seems that maybe here it is clear enough that this is *initial* data and the reactive computation will not react to later changes to it. But technically this is an opportunity to let staleness in.